.github/workflows/check-links.yml

```diff
@@ -0,0 +1,30 @@
+name: Scheduled jobs
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string.
+    # Run every day at 12:00AM UTC.
+    - cron:  '0 0 * * *'
+jobs:
+  buildSite:
+    env:
+      GOPATH: ${{ github.workspace }}
+    name: Install deps and build site
+    runs-on: ubuntu-latest
+    steps:
+      # Turnstyle is used to prevent multiple push jobs from
+      # running at the same time.
+      - name: Turnstyle
```

Comment:
You can remove this step here since the trigger for _this_ workflow is only a `cron` schedule.

Reply:
Ah, that's right! Thanks.